lib/permutations.rb

```diff
@@ -1,4 +1,22 @@
-
 def permutations?(string1, string2)
```

Comment:
You are going to need to count the number of occurrences of each character in each string.